### PR TITLE
added cloudlinux 6.6 support

### PR DESCRIPTION
**Pull request by wrender**
Hello,

I added some lines of code to support the latest version of CloudLinux 6.6 x86_64.  My apologies if I did not setup the pull request/repository correctly, as I am new to git.

Wes

### CODE REVIEW

File: rpm/setup [hunk 1]
```diff
@@ -57,6 +57,11 @@ DISTRO_PKG=$(rpm -q --whatprovides redhat-release)
 echo "+ uname -m"
 UNAME_ARCH=$(uname -m)
 
+echo "+ rpm -q --whatprovides cloudlinux-release"
+DISTRO_PKG=$(rpm -q --whatprovides cloudlinux-release)
```
Comment (wblankenship) [inline rpm/setup]: Should we be worried about replacing the value of `DISTRO_PKG` and `UNAME_ARCH`? I don't have a RHEL box to test on, but does `rpm -q --whatprovides x` fail when `x` is not provided? If so could we do

```
DISTRO_PKG=$(rpm -q --whatprovides redhat-release || rpm -q --whatprovides cloudlinux-release)
UNAME_ARCH=$(uname -m)
```

**Comment by wrender** [comment]
You're right wblankenship. My code is a bit sloppy.  I will make your recommended changes right now. :)

**Comment by wblankenship** [comment]
Not at all sloppy, it matches the format already there, and is well constructed :smile: overall this is awesome work.

**Comment by wrender** [comment]
Great!

File: rpm/setup [hunk 2]
```diff
@@ -52,11 +52,12 @@ fi
 #-check-distro-#
 
 ## Check distro and arch
-echo "+ rpm -q --whatprovides redhat-release"
-DISTRO_PKG=$(rpm -q --whatprovides redhat-release)
+echo "+ rpm -q --whatprovides redhat-release or rpm -q --whatprovides cloudlinux-release"
```
Comment (rvagg) [inline rpm/setup]: can this be `||` or is there a technical reason for "or", I'd like the `echo "+ ...` lines to reflect more accurately what's going on so users can see into the process

**Comment by rvagg** [comment]
So, clarify this for me: Cloud Linux is directly based on EL6 and has the same libc and all of that, but it doesn't have a `redhat-release` like the other EL6 variants but has an equivalent `cloudlinux-release`? I just want to make sure we're operating on the correct assumption that the base of this distro is the same as RHEL6 and CentOS 6 (because we can't / don't test beyond that ourselves).

**Comment by wrender** [comment]
CloudLinux 6.6 is a variant of CentOS 6.6.  There is a script that changes a CentOS install to CloudLinux, and from CloudLinux back to Centos.  The only things CloudLinux changes are the kernel, and adds a bunch of additional rpms for lve's that do not interfere with CentOS.  It's really just an "enhanced" version of CentOS designed for shared web hosting.

I think the "or" is needed for "cloudlinux-release" as that is the equivalent rpm for cloudlinux since it is actually a different distro.

**Comment by rvagg** [comment]
@wrender I'm referring to "or" vs "||" in the echo.

I'll do some tidying up before I push this live because we now need to look for [centos-release](https://github.com/nodesource/distributions/pull/15#issuecomment-70416242) since 6.6 there too, so there's some changes required.

**Comment by rvagg** [comment]
@wrender ok it's live at rpm.nodesource.com/setup, can you try it out and let me know please?

**Comment by wrender** [comment]
Ok. My branch/repository didn't seem to update with the merge/pull request.  Trying to update it now.

**Comment by wrender** [comment]
Works on CloudLinux 6.6 x86_64 without error.

**Pull request merged**